csrf/mod.ts

```diff
@@ -13,13 +13,30 @@ const primaryToken = createHash("sha512");
 primaryToken.update(v4.generate());
 const primaryTokenString = primaryToken.toString();
 
-export function CSRF() {
+type Options = {
+  cookie?: boolean;
+};
+
+const defaultOptions = {
+  cookie: false,
+};
+
+export function CSRF(options?: Options) {
+  if (!options) {
+    options = defaultOptions;
+  }
```

Comment:
I would change the signature to simplify it.
```typescript
type Options = {
  cookie?: boolean;
}

export function CSRF(options: Options = { cookie: false }) {
  // Do stuff. Options is already assigned with `cookie: false` when no variable is passed to the function.
}
```

Reply:
My dumb ass thought with that way, the user would still have to do `CSRF({})`, but i agree 👍

Reply:
Though i think keeping the `defaultOptions` would be good, keeps it scalable - thoughts?

Reply:
https://github.com/drashland/deno-drash-middleware/pull/57/commits/0285f7db1ab835b58c616b0de5c44524cd7b4dec, but should you disagree, i can always change it, just wanted to push changes up so they weren't only local

Reply:
Yours is better! Top notch stuff 👌🏿